docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -182,8 +182,8 @@ Before your test your build, ensure that you have installed and configured the f
     2. To view the `PID` of the active S3 service, run the command: `$ pgrep s3`
     3. To view the `PID` of the active Motr service, run the command: `$ pgrep m0`
 5. Follow these steps to install the AWS client and plugin:
-    1. To install the AWS client, use: `$ pip install awscli`
-    2. To install the AWS plugin, use: `$ pip install awscli-plugin-endpoint`
+    1. To install the AWS client, use: `$ pip3 install awscli`
+    2. To install the AWS plugin, use: `$ pip3 install awscli-plugin-endpoint`

```

Comment:
Add note here: If we install awscli n s3 development environment this way, Jenkins system tests will break due to version dependancy conflict in s3iamcli. Fix for this is in progress.

Reply:
fixed.